timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md

```diff
@@ -0,0 +1,393 @@
+ avg(block_fee_sat)*0.00000001 AS "mining fee"
+FROM one_hour_blocks
+WHERE bucket > now() - INTERVAL '5 day'
+GROUP BY bucket
+ORDER BY 1
+```
+![transactions in block and mining fee](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/tx_in_block_expensive.png)
+
+Unsurprisingly, there’s a high correlation between the number of transactions
+in a block and the mining fee. The more transactions a block has, the higher
+the block mining fee.
+
+Let’s dive deeper and see if there is the same correlation between block
+weight and mining fee. (Block weight is the size measure of a block). More
+transactions should increase the block weight, boosting the miner fee as well.
+This query looks very similar to the previous one:
+```sql
+SELECT
+ bucket as "time",
+ avg(block_weight) as "block weight",
+ avg(block_fee_sat*0.00000001) as "mining fee"
+FROM one_hour_blocks
+WHERE bucket > now() - INTERVAL '5 day'
+group by bucket
+ORDER BY 1
+```
+![block weight and mining fee](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/weight_fee.png)
+
+And yes—you can see the same kind of high correlation between block weight
+(defined in weight units) and mining fee. Except when the block weight gets
+close to its maximum value (4M weight units) in which case it's impossible for a
+block to include more transactions.
+
+In the previous charts, you saw how mining fees are correlated to block
+weights, transaction volumes, etc. But now, let’s analyze the data from a
+different perspective. Miner revenue is not only made up of miner fees, it
+also includes block rewards after mining a new block. This reward is currently
+6.25 BTC and it gets halved every four years. Let’s analyze miner revenue!
+
+### What percentage of the average miner's revenue comes from fees vs. block rewards?
+Miners are incentivized to keep the network up and running because they earn
+fees and rewards after mining each block. But let’s look at how much of their
+revenue comes from these two sources.
+
+```sql
+WITH coinbase AS (
+   SELECT block_id, output_total AS coinbase_tx FROM transactions
+   WHERE is_coinbase IS TRUE and time > NOW() - INTERVAL '5 days'
+)
+SELECT
+   bucket as "time",
+   avg(block_fee_sat)*0.00000001 AS "fees",
+   FIRST((c.coinbase_tx - block_fee_sat), bucket)*0.00000001 AS "reward"
+FROM one_hour_blocks b
+INNER JOIN coinbase c ON c.block_id = b.block_id
+GROUP BY bucket
+ORDER BY 1;
+```
+![fees and block reward](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/revenue_ratio.png)
+(THIS CHART NEEDS AN UPDATE)
+
+This chart analyzes the last 5 days of average miner revenue. The left
+axis is a percentage value that indicates the portion of total revenue that
+comes from transaction fees (green) and block rewards (yellow). It might be
+surprising for some, but most miner revenue comes from block rewards
+(6.25BTC at the moment). The fees portion never went above 3 % in the
+last 5 days. 
```

Comment:
```suggestion
This chart analyzes the last five days of average miner revenue. The left
axis shows the percentage of total revenue that
comes from transaction fees (green) and block rewards (yellow). Most miner
revenue actually comes from block rewards
(6.25&nbsp;BTC at the moment). Fees never accounted for more than 3% in the
last five days. 
```

Five or 5, doesn't really matter, just make it consistent. I've seen five everywhere else so went with that